### DISCUSSION TITLE
Disable schema-informations

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#4234] [Archive]
As discussed on [Discord](https://discord.com/channels/725371605378924594/725817819882586132/813555027137527829) there's a use to hide not needed database informations (the schema object) from the api.

As Rijk mentioned the app wouldn't work anymore.
That's why I'd suggest to combine that to the "app-access" option by default (only get the information if you've app-access). 
If those data are needed for users without app-access this could be overwritten via env variable.

**Comment by rijkvanzanten** [comment]
I think the preferred solution would be to allow configuring the individual schema fields through permissions, as you would with any other "normal" field in any other collection. This would allow you to add/remove any field you want in the API, while still allowing app-access to set a minimum required set 👍🏻

**Reply by Dominic-Marcelino** [reply]
Even better! But still would keep the default value set to none, as directus is also aiming to non technical users that otherwise would make unnecessary details public available

**Reply by rijkvanzanten** [reply]
Absolutely. All permissions defaults are none, so the same would apply to the schema info 🙂
